Documentation/DebugJavascriptDevTools.md

```diff
@@ -0,0 +1,29 @@
+# BabylonNative Script Debugging on Win32
+You can use Chromium DevTools to debug your JavaScript while running BabylonNative on Win32 (x86/x64) with the V8 javascript engine.
```

Comment:
>on Win32 (x86/x64) [](http://example.com/codeflow?start=83&length=18)

nit: on Win32 sounds kind of strange. Can this be "You can use chromium devtools to debug JavaScript while running a Win32 BabylonNative application (x86/x64) with the V8 JavaScript engine"?